pkg/controllers/clusterresourceplacementeviction/controller_test.go

```diff
@@ -31,6 +33,14 @@ const (
 	testEvictionName         = "test-eviction"
 )
 
+var (
+	validationResultCmpOptions = []cmp.Option{
+		cmp.AllowUnexported(evictionValidationResult{}),
+		cmpopts.IgnoreFields(metav1.ObjectMeta{}, "ResourceVersion"),
+		cmpopts.IgnoreFields(placementv1beta1.ClusterResourceBinding{}, "ResourceVersion"),
```

Comment:
This should be covered by the above ignore, cause "ResourceVersion" is part of metav1.ObjectMeta{}